cert-generation/main.go

```diff
@@ -50,66 +87,12 @@ func getSecureServiceSpec(namespace string) *corev1.Service {
 			Selector: label,
 		},
 	}
-
-	return service
-}
-
-func getSecureConfigMapSpec(namespace string) *corev1.ConfigMap {
-	return &corev1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      certConfigMapName,
-			Namespace: namespace,
-			Annotations: map[string]string{
-				"service.beta.openshift.io/inject-cabundle": "true",
-			},
-		},
+	_, err = clientset.CoreV1().Services(namespace).Create(service)
+	if err != nil {
+		os.Exit(1)
```

Comment:
figuring out why cert generation pod failed without any error logs could be a nightmare =)

Reply:
![Screenshot_20200609_155615](https://user-images.githubusercontent.com/5887312/84149871-cadd2500-aa69-11ea-9f15-7ab027291f96.png)